content/vault/v1.21.x/content/docs/license/index.mdx

```diff
@@ -23,6 +23,12 @@ Evaluation licenses have a 1-day grace period until the license expires. Once
 the evaluation license expires, Vault seals and cannot unseal until you apply
 a valid license.
 
+## PKI-only licenses
+
+PKI-only licenses restrict the secrets engines available on a cluster to Vault's
+built-in PKI secrets engine.
+Review the documentation on <a href="./license/pki">certificates-based entitlements</a>
+for additional information.
```

Comment:
```suggestion
PKI-only licenses restrict the available secrets engines on a Vault cluster to
the built-in PKI secrets engine.

Refer to the [PKI-only cluster overview](/vault/docs/license/pki) for additional
information.
```
Style correction: avoid possessives, prefer markdown links

Note: I used the current URL for the PKI overview in the suggestion, but (per my other comment) the overview doc doesn't belong in the `/license` directory